### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by caioaamaral**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+### 1. Installation from Packages:
+
+TODO
+
+    sudo apt-get install ros-indigo-...
+
+
+or you could also build this repository from source.
+
```
Comment (mateusmenezes95) [inline README.md]: Remove this for a while...

File: README.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+First, lets create a catkin workspace.
+
+    mkdir -p ~/catkin_ws/src
+
+**doogie_simulator** depends on [doogie_base] stack. So let's clone it inside our workspace source and install it.
+
+	cd ~/catkin_workspace/src
+	git clone http://github.com/doogie-mouse/doogie_base.git
+    rosdep install --from-paths src -ignore-src -r -y
```
Comment (mateusmenezes95) [inline README.md]: `rosdep install --from-paths src --ignore-src -r -y`

This command should be called from `~/doogie_ws`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+### 2. Building from Source:
+
+Attention, if you haven't installed [ROS] yet, please check [Ubuntu install of ROS Kinetic](http://wiki.ros.org/kinetic/Installation/Ubuntu). Desktop-Full Install is the recommended one in order to work with this repository.    
+
+**Building:**
+
+First, lets create a catkin workspace.
+
+    mkdir -p ~/catkin_ws/src
```
Comment (mateusmenezes95) [inline README.md]: `~/doogie_ws`

File: README.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+	git clone http://github.com/doogie-mouse/doogie_base.git
+    rosdep install --from-paths src -ignore-src -r -y
+
+Then, clone **doogie_simulator** also in your workspace source.
+        
+    git clone http://github.com/doogie-mouse/doogie_simulator.git
+
+Now, just build your catkin workspace.
+
+    cd ~/catkin_ws
```
Comment (mateusmenezes95) [inline README.md]: `~/doogie_ws`

File: README.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+    catkin build
```
Comment (mateusmenezes95) [inline README.md]: `catkin_make`

File: doogie_gazebo/README.md [hunk 6]
```diff
@@ -1,143 +1,151 @@
...
 
+   - **gazebo_ros_range** (plugin used here to simulate the IR sensors)
 
-#### Published Topics
+**gazebo_ros_control** and each of the **gazebo_plugins** used here in the simulation was previously declared in Doogie Mouse's URDF, check [doogie_description], and defined in [doogie_control].
 
-...
+# **Topics**
+
+- **`/move_base_controller/cmd_vel`** : [geometry_msgs/Twist]
```
Comment (mateusmenezes95) [inline doogie_gazebo/README.md]: Put the link to message description

File: README.md [hunk 7]
```diff
@@ -62,22 +61,23 @@ Attention, if you haven't installed [ROS] yet, please check [Ubuntu install of R
 
 First, lets create a catkin workspace.
 
-    mkdir -p ~/catkin_ws/src
+    mkdir -p ~/doogie_ws/src
 
 **doogie_simulator** depends on [doogie_base] stack. So let's clone it inside our workspace source and install it.
 
-	cd ~/catkin_workspace/src
+	cd ~/doogie_workspace/src
```
Comment (mateusmenezes95) [inline README.md]: `~/doogie_ws`

**Comment by caioaamaral** [comment]
Some more fixes or can we merge?

**Comment by mateusmenezes95** [comment]
> Some more fixes or can we merge?

No more issues for a while. I will merge

**Pull request merged**